_layouts/install.html

```diff
@@ -3,5 +3,5 @@
 ---
 {{ content }}
 <a href="/problem.html" class="help-button-holder">
-  <i class="fa fa-question fa-lg help-button"></i>
+  <i class="help-button" title="Hilfe bei der Installation">?</i>
```

Comment:
I am undecided between Hilfe and Probleme.

Reply:
Yeah, I was thinking about that too... I mean, technically neither is wrong. But if we go with "Probleme" I'd suggest going for "Probleme bei der Installation?" with a question mark at the end. Sounds better to me, imo.

Reply:
I guess no one will read the tool tip. So if we get weird mails, we can change it.